tests/admin_filters/tests.py

```diff
@@ -241,6 +242,15 @@ def setUp(self):
 
         self.request_factory = RequestFactory()
 
+        # Departments
```

Comment:
How about keeping setUp the same and making the necessary changes in the new test that needs it? I think that might make the expected test result easier to understand.